Web/www/index.js

```diff
@@ -0,0 +1,114 @@
+// ---------------------------------------------------------------------------
+// index.js
+// 
+// Author: Kevin McAndrew
+// Created: 4 Aug 2020
+// ---------------------------------------------------------------------------
+
+// ---------------------------------------------------------------------------
+// Create PID card
+// ---------------------------------------------------------------------------
+const pid_card = document.getElementById("pid_card");
+// Fetch JSON data
+let jsonData;
+fetch('pid.json')
+  .then(response => response.json())
+  .then(data => appendPID(data));
+
+function appendPID(data){
+  console.log(data);
+  jsonData = data;
+  // -------------------------------------------------------------------------
+  // -- Axis container
+  data.forEach((element) => {
+    // -----------------------------------------------------------------------
+    // -- Title
+    const title = `<br><p class="text--bold"> ${element["name"]}</p><br>`;
+    const titleDiv = document.createElement("div");
+    titleDiv.innerHTML = title;
+    pid_card.appendChild(titleDiv);
+    
+    // -----------------------------------------------------------------------
+    // -- Input and increment buttons
+    for (var key in element["data"]) {
```

Comment:
```suggestion
    Object.keys(element.data).forEach((key) => {
      ...
    });
```
This is a more functional way of getting the keys for each element in the dictionary. One thing to note is that unlike in Python, dictionaries in Javascript are just objects so you can also use `element.data` rather than having to reference like a Python dictionary each time.